app/pages/project/index.cjsx

```diff
@@ -179,10 +179,17 @@ ProjectPage = React.createClass
       @state.activeWorkflows[randomIndex].id
 
   getWorkflow: (selectedWorkflowIndex) ->
-    @setState {
-      selectedWorkflow: @state.activeWorkflows[selectedWorkflowIndex],
-      loadingSelectedWorkflow: false
-    }
+    apiClient.type('workflows').get({ id: "#{@state.activeWorkflows[selectedWorkflowIndex].id}" })
+      .catch (error) =>
+        console.error error
+        @setState {
+          loadingSelectedWorkflow: false
+        }
+      .then ([workflow]) =>
+        @setState {
```

Comment:
I can see checking for `@state.selectedWorkflow` to prevent unnecessary API calls, but I'm not sure if I understand the utility of storing the full resource in `state.activeWorkflows` too. It's used for two reasons: to check if the selected workflow is active (for now, but the project resource now only returns active workflows links! zooniverse/Panoptes#2073) and to use with rendering the buttons on the home page. Neither use needs the full resource, imo.